js/layout-javascript/news-feed-code.js

```diff
@@ -165,6 +167,40 @@ DynamicList.prototype.attachObservers = function() {
         console.error(error);
       });
     })
+    .on('click', '.sort-group .list-sort li', function (e) {
+      e.stopPropagation();
+
+      var $sortListItem = $(e.currentTarget);
+      var oldSortOrder = $sortListItem.data('sortOrder');
+      var $sortOrderIcon = $sortListItem.find('i');
+      var $sortList = _this.$container.find('.list-sort li');
+      var sortClasses = {
+        'none': 'fa-sort',
```

Comment:
Single quote not needed.

```suggestion
        none: 'fa-sort',
```